utils/build.ps1

```diff
@@ -2439,6 +2439,17 @@ function Test-Format {
   }
 }
 
+function Build-LMDB() {
+  Build-SPMProject `
```

Comment:
Should this be `Build-CMakeProject`? This doesn't match the commit message. Do you have numbers for what the toolchain size impact is on static vs dynamic builds?

Reply:
Good catch! Admittedly I'm quite new to this and haven't been able to compile Swift on Windows at all. So, I don't have those numbers right now.